documentation/KnownChanges.md

```diff
@@ -199,6 +199,10 @@ This rule is disabled by default in StyleCop Analyzers, but can be enabled by us
 :warning: StyleCop Analyzers does not report SA1305 for parameters in overriding methods and methods which implement an
 interface. StyleCop Classic reported SA1305 for all methods.
 
+### SA1313
+
+StyleCop Analyzers treats allows the double underscore `__` as a lambda parameter name as well as the single underscore.
```

Comment:
📝 This document is for changes relative to StyleCop Classic. If I'm not mistaken, that library will ignore parameters with any number of underscores. The difference in StyleCop Analyzers is we only make the exception for one or two.

Reply:
I wasn't sure so I left the wording ambiguous :D Good to know.